apis/keda/v1alpha1/scaledobject_types.go

```diff
@@ -109,6 +111,8 @@ type ScaledObjectSpec struct {
 type Fallback struct {
 	FailureThreshold int32 `json:"failureThreshold"`
 	Replicas         int32 `json:"replicas"`
+	// +optional
+	Behavior string `json:"behavior,omitempty"`
```

Comment:
I think we can leverage kubebuilder validation tags
```suggestion
	// +optional
	// +kubebuilder:validation:Enum=static;useCurrentReplicasAsMinimum
	Behavior string `json:"behavior,omitempty"`
```
together with `optional` it should allow `""` and the two `"static"` or `"useCurrentReplicasAsMin"`

and the default error message looks very readable imho
```
The ScaledObject "http-server" is invalid: spec.fallback.behavior: Unsupported value: "xyz": supported values: "static", "useCurrentReplicasAsMinimum"
```